runtime/browser/xwalk_render_message_filter.cc

```diff
@@ -50,6 +55,26 @@ void XWalkRenderMessageFilter::OnSubFrameCreated(
   XWalkContentsIoThreadClient::SubFrameCreated(
       process_id_, parent_render_frame_id, child_render_frame_id);
 }
+
+void XWalkRenderMessageFilter::OnShouldOverrideUrlLoading(
```

Comment:
This function was basically copied from Chromium, yet there's no copyright attribution.